qualtran/bloqs/qft/qft_phase_gradient.py

```diff
@@ -18,24 +18,23 @@
 import cirq
 from numpy.typing import NDArray
 
-from qualtran import GateWithRegisters, QFxp, QUInt, Signature
+from qualtran import bloq_example, BloqDocSpec, GateWithRegisters, QFxp, QUInt, Signature
 from qualtran.bloqs.arithmetic.multiplication import PlusEqualProduct
 
 
 @attrs.frozen
 class QFTPhaseGradient(GateWithRegisters):
-    r"""QFT implemented using phase gradient trick. Uses O(n**2) T-gates for an n-bit register.
-
-    Given an n-bit phase gradient state $|\phi\rangle$ prepared as
+    r"""QFT implemented using coherent addition into a phase gradient register
 
+     A variant of the Quantum Fourier Transform (QFT) that utilizes an additional register provided in a phase gradient state to switch controlled rotations to coherent additions. Given an ancilla register prepared in the state
```

Comment:
Each individual line should be <= 100 characters long. We should probably start enforcing this as part of the pylint (cc @mpharrigan) but this has been part of our style guide. Can you please add some line breaks to make sure we don't have lines with > 100 characters in the docstrings? This gets taken care of for code automatically as part of the auto formatting.